Dev/VideoChatClient/PacketUtils/PacketUtils.hpp

```diff
@@ -32,11 +33,13 @@ namespace PacketUtils
 		uint32_t meeting_id; // 4 bytes
 		uint32_t user_unique_id; // 4 bytes
 		char name[PACKET_HEADERS_NAME_SIZE]; // 64 bytes
-		PacketTypes data_type; // 2 byte
+		PacketTypes packet_type; // 2 byte
 		uint32_t data_size; // 4 bytes
 	};
 
 #pragma pack() // Continue using the normal padding.
 
-	void validate_packet(const PacketHeaders& packet_headers, uint32_t meeting_id);
+	void validate_packet(const PacketHeaders& packet_headers, uint32_t meeting_id,
+	                     PacketTypes packet_type = PacketTypes::Uninitialized);
+	void validate_packet(const PacketHeaders& packet_headers, PacketTypes packet_type = PacketTypes::Uninitialized);
```

Comment:
packet_type should not be auto initialized, because validate_packet checks it's not 0, meaning you want the user to pass it on every call. fix both🐱‍🏍

Reply:
I don't think you understand what I was trying to do here., My goal was to allow the user to validate only one or some fields, and not all at once

Reply:
e.g.
validate_packet(headers); // just validate magic
validate_packet(headers, meeting_id); //validate magic and meeting_id
validate_packet(headers, meeting_id, packet_type); //validate magic and meeting_id and packet_type
validate_packet(headers, packet_type); //validate magic and packet_type